pages/index.js

```diff
@@ -16,6 +16,10 @@ function Home({ data: initialData = {}, preview, token }) {
   const pageData = data?.page || data?.[0];
   const slug = pageData?.slug;
 
+  useEffect(() => {
+    localStorage.setItem("preview", preview);
```

Comment:
Hi Ronix,

I think we can make use of the preview prop value to check if the current page is in preview mode or not.

You can try to verify if we're on preview mode through the Open Preview from the studio or add the /api/preview?secret=${secret}&slug=${slug} to the site URL.

To test the live URL (not in preview mode), you can click the live URL badge or access the URL directly from chrome incognito mode.

What do you think?